internal/driver/glfw/window_darwin.go

```diff
@@ -0,0 +1,23 @@
+//go:build darwin
+
+package glfw
+
+import (
+	"fyne.io/fyne/v2/driver"
+)
+
+// assert we are implementing driver.NativeWindow
+var _ driver.NativeWindow = (*window)(nil)
+
+func (w *window) RunNative(f func(any) error) error {
+	var err error
+	done := make(chan struct{})
```

Comment:
Do we need the done-channel here? It seems like `runOnMain` already waits as long as the driver already is started. We are waiting twice in most cases. Will this functionality make any sense in the case when the driver hasn't started (we won't have any window)?

Reply:
Removed it